docs/reference/commandline/dockerd.md

```diff
@@ -906,10 +904,6 @@ scrape_configs:
       - targets: ['127.0.0.1:9323']
 ```
 
-Please note that this feature is still marked as experimental as metrics and metric
-names could change while this feature is still in experimental.  Please provide
-feedback on what you would like to see collected in the API.
-
```

Comment:
We may want to keep some wording about this.


The TL;DR is that we removed the experimental gate on the daemon side for the metrics socket, because we saw no strong motivation to require _both_ the experimental option to be set, _and_ setting the option for the metrics-address.

That said, naming of metrics may still change (also see https://github.com/moby/moby/issues/27307); an initial set of metrics implemented, and the original plan was to add / change those based on feedback (we didn't get a lot of that, and I don't think here's been much activity around it).

Reply:
I'd say fine to remove this. The docs shouldn't discuss things that may or may not happen in the future.